data/indices/h3-index/index.html

```diff
@@ -0,0 +1,103 @@
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="H3SpatialIndex">H3 Spatial Index</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <img class="inline-text-left outline" src="{% link images/h3_single.png %}" alt="H3 Index Sample"
+      loading="lazy" />
+    <p>The <a href='https://h3geo.org/'>H3 Spatial Index</a> (created by Uber) helps users aggregate point data using an easily reusable common grid. This allows you to maintain a consistent geometry across all your analyses, which makes them easier to compare. H3 uses highly-optimized math routines to assign lat/long coordinates an ID corresponding to a polygon (almost always hexes) at a given resolution rather than performing a much slower point-in-polygon geometrical operation. Once you've generated IDs for your points, you can use these polygons to aggregate them and display summary statistics on a map.</p>
+    <p>We used the H3 library to create the index polygons that cover the entire State of Utah at resolutions 5 through 9. You can find more information about how we created them, along with an example of how to use them, in our <a href='{% link _posts/2022-10-26-using-h3-hexes.md %}'>H3 blog post</a>.</p>
+    <p>Like the rest of our SGID data, the spatial reference for these polygons is UTM Zone 12N, NAD83. However, remember that your points must be in WGS 84 lat/long in order to use the H3 API to assign them a hex ID. The polygons are copyright 2022 Uber Technologies, Inc and licensed under version 2.0 of the <a href='http://www.apache.org/licenses/LICENSE-2.0'> Apache License</a>.</p>
+    <h5>Related Resources</h5>
+    <ul class="dotless">
+      <li><a href="https://h3geo.org/docs/core-library/restable/">Table of hexagon dimensions for different resolutions</a>
+      <li><a href='{% link _posts/2022-10-26-using-h3-hexes.md %}'>Our H3 blog post</a>
+    </ul>
+    <p>{% include contact.html subject=page.title contact=site.data.contacts.jacob %}</p>
+  </div>
+  <div class="grid__col grid__col--1-of-2">
+    <div class="panel">
+      {% include fa_icon.html download=true lower=true class="svg-inline--fa fa-w-16 fa-pull-right fa-2x" %}
+      <h5 id="UtahParcels-downloads">Downloads</h5>
+    </div>
+    <div class="panel-content">
+      <table id="UtahParcels-dlist-table">
+        <tr>
+          <th class="text-left">Resolution</th>
+          <th class="text-left">SGID Open Data</th>
+          <th class="text-left">File Geodatabase</th>
+          <th class="text-left">Shapefile</th>
+        </tr>
+        <tr>
+          <td>Level 5</td>
+          <td><a href="https://opendata.gis.utah.gov/datasets/utah-h3-hexes-level-5">Downloads and APIs</a></td>
+          <td><a href="https://opendata.gis.utah.gov/datasets/980d3af37717456b92d7039d9ffa35f3_0.gdb">Download</a></td>
+          <td><a href="https://opendata.gis.utah.gov/datasets/980d3af37717456b92d7039d9ffa35f3_0.zip">Download</a></td>
+        </tr>
```

Comment:
**suggestion**: It would be really great to add the [open sgid table names](https://github.com/agrc/gis.utah.gov/blob/master/_includes/opensgid.html) for these. It's not common with the old data sets but I'd like to see this feature get used more.

Reply:
I added the table names, but the new width messes with the updates column. Since the data are static, I just removed the update column completely.

Reply:
Yah that's fine. The table is not great on mobile or small screens but I think it's safe to assume not many people will be downloading these on those devices.